package.json

```diff
@@ -42,7 +42,8 @@
     "ember-cli-deploy"
   ],
   "ember-addon": {
-    "configPath": "tests/dummy/config"
+    "configPath": "tests/dummy/config",
+    "defaultBlueprint": "deploy-config"
```

Comment:
I renamed the blueprint to `deploy-config` so the command reads `ember g deploy-config`.  I thought it was better than `ember g ember-cli-deploy`.